src/pages/meetings.js

```diff
@@ -77,7 +80,9 @@ const Meetings = () => {
 							?.filter(meeting => meeting.privacy !== "private" || audits || user?.isFaculty)
 							.map(meeting => {
 								const newMeeting = { ...meeting };
-								newMeeting.title = meeting.organization.name + " - " + meeting.title;
+								const meetingRoom =
+									typeof meeting.rooms[0] === "undefined" ? "" : `: Room ${meeting.rooms[0].name}`;
```

Comment:
should not be in a string? anyway test again make sure I'm not making stuff up

Reply:
I did `console.log(meeting.rooms[0]);` and got this:
![image](https://user-images.githubusercontent.com/25336089/147396298-653acf5a-4ac1-478f-b715-d59492ebc7d5.png)

so `meeting.rooms[0].name` is indeed a string if `meeting.rooms[0]` is defined.

Reply:
And this is an image of what the calendar looks like with this PR: 
![image](https://user-images.githubusercontent.com/25336089/147396335-2ae9c8a6-2881-4d33-a3d3-6fde547bbfed.png)
![image](https://user-images.githubusercontent.com/25336089/147396343-048d4a37-5cf8-4ff2-9068-e6d08d34a010.png)